HISTORY.md

```diff
@@ -10,6 +13,7 @@
     methods/hoeffding_trees/information_gain.hpp (#2556).
 
   * Added macro for changing stream of printing and warnings/errors (#2556).
+>>>>>>> origin/master
```

Comment:
```suggestion
```

Reply:
Thanks!  How did I miss that during the merge?  Oops.